tests/System/Routes/CancelDonationRouteTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WMDE\Fundraising\Frontend\Tests\System\Routes;
+
+use WMDE\Fundraising\Frontend\Tests\System\WebRouteTestCase;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class CancelDonationRouteTest extends WebRouteTestCase {
+
+	public function testGivenValidArguments_requestResultsIn200() {
+		$client = $this->createClient();
+
+		$client->request(
+			'POST',
+			'/donation/cancel',
+			[
+				'sid' => '',
+				'token' => '',
+				'utoken' => '',
+			]
+		);
+
+		$this->assertSame( 200, $client->getResponse()->getStatusCode() );
+	}
+
+	public function testGivenGetRequest_resultIs405() {
```

Comment:
Maybe name it `testGivenGetRequest_resultIsMethodNotAllowed`. i had to think of which status code this is.